### DISCUSSION TITLE
Copilot Individual as student suddenly stopped working inside zed? Copilot in VSCode still works.

### DISCUSSION THREAD

**Discussion by ilukasschneider** [zed-industries/zed#11354] [Feature Requests]
Greetings, I fell in love with zed a couple of weeks ago and started using copilot. Worked fine but today I lost my copilot and I really miss him 🥲

![Bildschirmfoto 2024-05-03 um 15 59 31](https://github.com/zed-industries/zed/assets/18196954/e5b301a7-f80f-4572-99ed-e9efce37ddc2)

This is the Error I get. I tried using Copilot in VSCode and it works fine. I logged off and logged in again, disabled and enabled Copilot and even reinstalled zed - nothing seems to work.

Does have somebody have an idea on how to bring back my copilot?

![Bildschirmfoto 2024-05-03 um 16 09 48](https://github.com/zed-industries/zed/assets/18196954/0d5bbf31-9932-4452-b9ca-95e48d006ea8)

**Comment by bennetbo** [comment]
Hey,
are you talking about copilot or the assistant here? These are two separate things.
- The inline completions you see in the editor come from Github Copilot
- The assistant uses OpenAI API to generate responses

Meaning those are two different services, one needs a valid GitHub copilot license and one needs a valid OpenAI API key

**Reply by ilukasschneider** [reply]
Hey, the inline completions seem to work but the assistant doesnt :(

Ok this is my bad - I thought its both copilot and I forgot I have put in my api key at some point - thank you!
